frontend/src/config.ts

```diff
@@ -18,6 +18,8 @@ export const EnvSchema = z.object({
     }),
     api: z.object({
         baseUrl: z.string(),
+        roundRobinUrl: z.string(),
```

Comment:
Can you add the values to the .env files?

Reply:
yup